src/Synapse/Synapse/help/Restore-AzSynapseSqlPool.md

```diff
@@ -64,6 +78,33 @@ PS C:\> $restoredPool = Restore-AzSynapseSqlPool -FromRestorePoint -RestorePoint
 
 This command creates an Azure Synapse Analytics SQL pool by leveraging a restore point from any existing SQL pool to recover or copy from a previous state.
 
+### Example 2
+```powershell
+PS C:\> # Transform Synapse SQL pool resource ID to SQL database ID because
+PS C:\> # currently the command only accepts the SQL databse ID. For example: /subscriptions/<SubscriptionId>/resourceGroups/<ResourceGroupName>/providers/Microsoft.Sql/servers/<WorkspaceName>/recoverabledatabases/<DatabaseName>
+PS C:\> $pool = Get-AzSynapseSqlPoolGeoBackup -ResourceGroupName ContosoResourceGroup -WorkspaceName ContosoWorkspace -Name ContosoSqlPool
+PS C:\> $databaseId = $pool.Id -replace "Microsoft.Synapse", "Microsoft.Sql" `
+PS C:\>     -replace "workspaces", "servers"
+PS C:\> 
+PS C:\> # Restore to same workspace with source SQL pool
+PS C:\> $restoredPool = Restore-AzSynapseSqlPool -FromBackup -TargetSqlPoolName ContosoRestoredSqlPool -ResourceGroupName $pool.ResourceGroupName -WorkspaceName $pool.WorkspaceName -ResourceId $databaseId
+```
+
+This command creates an Azure Synapse Analytics SQL pool which restores from the SQL pool backup.
+
+### Example 3
+```powershell
+PS C:\> # Transform Synapse dropped SQL pool resource ID to SQL pool resource ID
+PS C:\> $pool = Get-AzSynapseDroppedSqlPool -ResourceGroupName ContosoResourceGroup -WorkspaceName ContosoWorkspace -Name ContosoSqlPool
+PS C:\> $poolId = $pool.Id.Split(",")[0]
```

Comment:
what does `$pool.Id` look like? Why do you need to split it by comma

Reply:
/subscriptions/<SubscriptionId>/resourceGroups/<ResourceGroupName>/providers/Microsoft.Synapse/workspaces/<WorkspaceName>/restorableDroppedSqlPools/sqlpool,132767676155800000

Above is an example of pool id. It's a `restorableDroppedSqlPools` with ",132767676155800000" which is deletion time. According our discuss in Teams, sourceDatabaseId can not be of this type, it should either be of type Microsoft.Synapse/workspaces/sqlPools or Microsoft.Sql/servers/databases."

Reply:
Thanks for the explanation.